Makefile

```diff
@@ -129,3 +129,7 @@ test: dist/main.js $(BUILD)/test.out
 .PHONY: watch
 watch: $(BOWER_COMPONENTS) $(NODE_MODULES)
 	npx watch-exec --command 'make dist/main.js' --watch $(SRC)
+
+.PHONY: serve
+serve:
+	http-server ./dist -p 80
```

Comment:
Can we also take the port as an argument (defaulting to `80`)? If someone is running `make serve` outside of the k8s workflow (not as root), it's likely not going to work.